client/src/views/clan/ClanThreadView.tsx

```diff
@@ -177,7 +177,7 @@ const ClanThreadView = () => {
                       <Box component="img" src="/images/clan/master.gif" sx={{ ml: 1, width: 7 }} />
                     )}
                   </Box>
-                  <Text color="primary">{moment(post.date).format('D MMM YYYY HH:mm')}</Text>
+                  <Text color="primary">{dayjs(post.date).format('D MMM YYYY HH:mm')}</Text>
```

Comment:
Missing utc in the original code

```suggestion
                  <Text color="primary">{dayjs.utc(post.date).format('D MMM YYYY HH:mm')}</Text>
```